### DISCUSSION TITLE
Add default value for flags in help message

### DISCUSSION THREAD

**Discussion by zupat** [vlang/v#12695] [Ideas]
app.add_flag(cli.Flag{
		flag: .bool
		name: 'verbose'
		default_value: ['false']
		description: 'verbosely list files processed being processed'
	})
it would be useful to include the default value in the help message

    Flags:
    -verbose          verbosely list files being processed **(defaults to false)**<- this should be included
    -help             Prints help information.

    Commands:
    help                Prints help information.

**Comment by MaxPaw** [comment]
There is (required) if option is required, seems like (default: 123) should be added as well
